packages/@o3r/schematics/src/utility/package-manager-runner.ts

```diff
@@ -64,25 +67,45 @@ export function getPackageManager(options?: PackageManagerOptions) {
 
 /**
  * Get command to run scripts with your package manager
- *
  * @param workspaceConfig Workspace configuration
+ * @param packageName Name of the package of the workspace to run the script
```

Comment:
maybe more explicit doc to not a have a confusion with the name of the project in angular.json
'Name of the package of the workspace to run the script (name from package.json)'